src/http/index.ts

```diff
@@ -22,19 +23,14 @@ const DEFAULT_OPTIONS: TidewaveConfig = {
   host: 'localhost',
 } as const;
 
-export interface TidewaveConfig {
-  allowRemoteAccess?: boolean;
-  allowedOrigins?: string[];
-  port?: number;
-  host?: string;
-}
-
 export function configureServer(
   server: Server = connect(),
   config: TidewaveConfig = DEFAULT_OPTIONS,
 ): Server {
   const securityChecker = checkSecurity(config);
 
+  server.use(`${ENDPOINT}`, CSPMiddleware);
+  server.use(`${ENDPOINT}`, logger);
```

Comment:
Same here: If those will appear in the log by default, we definitely want to remove it. :) In all other integrations we silence Tidewave logs (or perhaps we enable it only if an env var is set).